mobile-save-for-later/riff-raff.yaml

```diff
@@ -7,8 +7,6 @@ deployments:
     parameters:
       bucket: mobile-dist
       functionNames:
-        - mobile-save-for-later-SAVE-
```

Comment:
We are deleting these Lambdas in this PR, so we no longer want to (attempt to) update their application code.